GrandCentralBoard/Widgets/WebsiteAnalytics/PageViewsSource.swift

```diff
@@ -47,12 +47,19 @@ final class PageViewsSource: Asynchronous {
             return
         }
 
+        let validPathPrefix = self.validPathPrefix
         dataProvider.fetchPageViewsReportFromDate(reportStartTime, toDate: now) { result in
-            let pathToTitleTranslator = self.pathToTitleTranslator
             switch result {
-            case .Success(let reports):
-                let pageViews = reports.rows.flatMap { PageViewsRowReport(analyticsReportRow:$0, pathToTitleTranslator: pathToTitleTranslator) }
-                return closure(.Success(pageViews))
+            case .Success(let report):
+                let pageViews = report.rows.flatMap { row -> PageViewsRowReport? in
+                    let pageViewsReport = PageViewsRowReport(analyticsReportRow:row)
+                    if let pageViewsReport = pageViewsReport, let prefix = validPathPrefix where !pageViewsReport.hasTitleWithPrefix(prefix) {
```

Comment:
It will not work with guard, because `validPathPrefix` is optional, and it should be checked only if it is not nil. You can check cause I added some tests :) (Or maybe you have something else in mind than just swapping `if` for `guard`, let me know)